src/Blog/NewsBundle/Controller/IndexController.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Blog\NewsBundle\Controller;
+
+use Blog\NewsBundle\Entity\Advertisement;
+use Blog\NewsBundle\Entity\Contact;
+use Blog\NewsBundle\Form\ContactType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class IndexController extends Controller
+{
+    /**
+     * @Template("BlogNewsBundle:Index:index.html.twig")
+     * @return array
+     */
+    public function indexAction()
+    {
+        $categoryTitle = $this->get('category_title');
+        $categories = $categoryTitle->getShortTitles();
+
+        return array(
+            'name' => 'green',
+            'categories' => $categories
```

Comment:
in multi-line arrays add comma after the last array element (it allows easier addition of new elements at the end)